gax/src/main/java/com/google/api/gax/batching/BlockingSemaphore.java

```diff
@@ -70,4 +77,33 @@ public synchronized boolean acquire(long permits) {
     }
     return true;
   }
+
+  public synchronized boolean acquirePartial(long permits) {
```

Comment:
since the class is not public, method needn't be either

Reply:
Hmm, I don't think it can compile if the methods are public: "attempting to assign weaker access privileges ('package-private'); was 'public" because "All abstract, default, and static methods in an interface are implicitly public, so you can omit the public modifier." (https://docs.oracle.com/javase/tutorial/java/IandI/interfaceDef.html). I also tried accessing the class from other projects and confirmed it's not visible.

Reply:
Sorry, I meant the code won't compile if the methods **aren't** public.

According to https://docs.oracle.com/javase/tutorial/java/IandI/interfaceDef.html, "All methods in an interface are implicitly public", so the overrides need to be public too.

Reply:
OK, so @Override is missing. Please add.